qa-tests-backend/src/test/groovy/PolicyFieldsTest.groovy

```diff
@@ -18,10 +18,13 @@ import org.junit.Assume
 import org.junit.experimental.categories.Category
 import services.AlertService
 import services.PolicyService
+import spock.lang.IgnoreIf
 import spock.lang.Shared
 import spock.lang.Unroll
 import util.Env
 
+// TODO(ROX-12814): Scanner OOMs on this test in some Openshift jobs.
+@IgnoreIf({ Env.mustGetOrchestratorType() == OrchestratorTypes.OPENSHIFT })
```

Comment:
What about only disabling for OSD GCP? Other openshifts do not have this OOM IIRC?
```suggestion
@IgnoreIf({ Env.CI_JOBNAME.contains("osd-gcp") })
```

Reply:
There are mentions of failures on `rosa` and `openshift-oldest` too in the ticket :-/